backend/app/emails.py

```diff
@@ -136,11 +136,12 @@ def send_email(info: EmailInfo, db):
                     messages.append(_create_message(user, info, db))
 
         if info.inform_only_moderators or info.inform_moderators:
-            moderators = db.session.query(models.FlathubUser).filter_by(
-                is_moderator=True
-            )
-            for user in moderators:
-                messages.append(_create_message(user, info, db))
+            moderator_role = models.FlathubUser.by_role(db, "moderator")
```

Comment:
We will have to explicitly give the moderator role (in addition to `admin`) for people to get these mails now.